extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/styles.scss

```diff
@@ -158,10 +158,9 @@
   display: flex;
   flex-direction: column;
   border-radius: 0.5rem;
-  border: 1px solid var(--sds-clr-gray-01);
+  border: 1px solid var(--sds-clr-gray-06);
   position: relative;
-  margin: -1.5rem 0 1.5rem 0;
-  padding-top: #{pxToRem(24px)};
```

Comment:
this styling choice was cutting off 1px from the top of the Card. I think this had been done early on to deal with some other styling wonkiness, but changing this to the more standard `margin: 0 0 24px 0` does not have any adverse results